internal/output/plain.go

```diff
@@ -217,14 +238,21 @@ func sprintSlice(value interface{}) (string, error) {
 
 	result := []string{}
 	for _, v := range slice {
-		stringValue, err := sprint(v)
+		vv := v
+		if vp, ok := v.(interface{ Value() interface{} }); ok {
+			vv = vp.Value()
+		}
+
+		stringValue, err := sprint(vv)
 		if err != nil {
 			return "", err
 		}
 
 		// prepend if stringValue does not represent a slice
-		if !isSlice(v) {
-			stringValue = " - " + stringValue
+		if vp, ok := v.(interface{ PrependString() string }); ok {
+			stringValue = vp.PrependString() + stringValue
+		} else if !isSlice(v) {
+			stringValue = "• " + stringValue
```

Comment:
These are all interesting solutions to a problem we need to solve; but not in this story. This warrants further discussion. I don't want to just accept this to unblock the story without giving this some proper thought first. And as I said on Slack this is ripe for a refactor, so rather than adding more throwaway code that's going to make it harder to replace I'd rather we just use the tools we have.

Reply:
Okay, I have filed a story for this: https://www.pivotaltracker.com/story/show/176127475